recipes/fenics-basix/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "fenics-basix" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}-meta
+  version: {{ version }}
+
+source:
+  # 0.4.1 isn't on PyPI
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/fenics/basix/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 34de61ca3e39b1aa0392ec6dad292bb9ffaf965c0734a8638427b9d850180dc7
+
+build:
+  skip: true  # [win]
+  number: 0
+
+outputs:
+  - name: fenics-libbasix
```

Comment:
Is any c developer going to link against libbasix? If so, there should be a build/run_export with appropriate pinning and an existence check for one of the headers.

Reply:
Confirmed with maintainers that x.x.x is the right choice for libbasix.